### ISSUE TITLE
support "exclude" configuration directive after #17

### ISSUE THREAD

**Issue by jtimberman** [test-kitchen/test-kitchen#29]
The current behavior of test-kitchen allows exclusions in the Kitchenfile based on platform.

Jamie's current behavior allows use of regular expressions to limit the platforms to run, but it would be nice to have certain platforms automatically excluded from a configuration or "suite". Also, specific versions should be excluded:

http://tickets.opscode.com/browse/KITCHEN-46
